pytorch_lightning/profiler/xla.py

```diff
@@ -12,32 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """XLA Profiler will help you debug and optimize training workload performance for your models using Cloud TPU
```

Comment:
maybe move this inside the class definition for consistency with other components.

Reply:
Updated!